cmd/config/crawler/crawler.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package crawler
+
+import (
+	"errors"
+
+	"github.com/minio/minio/cmd/config"
+)
+
+// Compression environment variables
+const (
+	BitrotScan = "bitrot"
```

Comment:
can the variable be called `bitrotscan` such that ENV turns out to be MINIO_CRAWLER_BITROTSCAN instead of MINIO_CRAWLER_BITROT ?

Reply:
Is there an env? I didn't really want that. From the other code it seemed like you'd have to add it manually.

Changing the var name either way.

Reply:
You have no choice, all config options have ENV values automatically ..